opentracing-noop/pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
-        <main.java.version>1.6</main.java.version>
+        <main.java.version>1.7</main.java.version>
```

Comment:
Since noop will be required for the GlobalTracer and I want to use GlobalTracer for stagemonitor, I'm not in favour of that. What is api-module?